firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/ImagenInlineImage.kt

```diff
@@ -1,12 +1,37 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package com.google.firebase.vertexai.type
 
 import android.graphics.Bitmap
 import android.graphics.BitmapFactory
 import android.util.Base64
 
+/**
+ * Represents an Imagen Image that is contained inline
+ *
+ * @param data Contains the raw bytes of the image
+ * @param mimeType Contains the MIME type of the image (for example, `"image/png"`)
```

Comment:
question on this... are the double-quotes needed on image/png?  If we include them, then it represents the ACTUAL value/format the dev needs to provide.  Do we need to go into that level of detail about the syntax in their code?

Reply:
Its the other way around, this is an example of the ACTUAL value that the dev can expect to get when calling the getter.